### PR TITLE
Fixed quotes to represent intent of examples

### PR DESCRIPTION
**Pull request by xdfil**
I think the changes I made are the true intent of the author

+label: docsite_pr

##### ISSUE TYPE

 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by samdoran** [comment]
Thank you for your contribution. Unfortunately, the changes made would not work in the second example.

Double quotes and single qutoes in YAML have different escaping requirements that behave similar to other languages. From the [YAML spec](http://yaml.org/spec/1.2/spec.html#id2787109) on double-quoted style:

>This is the only style capable of expressing arbitrary strings, by using “\” escape sequences. This comes at the cost of having to escape the “\” and “"” characters. 

So any backrefs would have to escape the backslah, e.g., `\\1Xms`

The single-quoted style interprets pretty much everything as a literal, with only single quotes having to be escpaed. From the [YAML spec](http://yaml.org/spec/1.2/spec.html#id2788097) on single-quoted style:

>The single-quoted style is specified by surrounding “'” indicators. Therefore, within a single-quoted scalar, such characters need to be repeated. This is the only form of escaping performed in single-quoted scalars. In particular, the “\” and “"” characters may be freely used.

That's why it's generally easier to use single quotes for regular expressions in YAML.

File: lib/ansible/modules/files/lineinfile.py [hunk 1]
```diff
@@ -158,26 +158,26 @@
...
     regexp: '^%wheel\s'
-    line: '%wheel ALL=(ALL) NOPASSWD: ALL'
+    line: "%wheel ALL=(ALL) NOPASSWD: ALL"
 
 # Yaml requires escaping backslashes in double quotes but not in single quotes
 - lineinfile:
     path: /opt/jboss-as/bin/standalone.conf
     regexp: '^(.*)Xms(\\d+)m(.*)$'
-    line: '\1Xms${xms}m\3'
+    line: "\1Xms${xms}m\3"
```
Comment (samdoran) [inline lib/ansible/modules/files/lineinfile.py:172]: This example is supposed to show the difference in single vs double quoting requirements in YAML, but it's incorrect. This like should be singel quoted but the `regexp` should be double quoted.

File: lib/ansible/modules/files/lineinfile.py [hunk 2]
```diff
@@ -158,26 +158,26 @@
...
-# Fully quoted because of the ': ' on the line. See the Gotchas in the YAML docs.
+# Double quoted because of the ': ' on the line. See the Gotchas in the YAML docs.
 - lineinfile:
     path: /etc/sudoers
     state: present
     regexp: '^%wheel\s'
-    line: '%wheel ALL=(ALL) NOPASSWD: ALL'
+    line: "%wheel ALL=(ALL) NOPASSWD: ALL"
 
 # Yaml requires escaping backslashes in double quotes but not in single quotes
```
Comment (samdoran) [inline lib/ansible/modules/files/lineinfile.py:168]: Capitalize YAML

File: lib/ansible/modules/files/lineinfile.py [hunk 3]
```diff
@@ -158,26 +158,26 @@
     line: '192.168.1.99 foo.lab.net foo'
     create: yes
 
-# Fully quoted because of the ': ' on the line. See the Gotchas in the YAML docs.
+# Double quoted because of the ': ' on the line. See the Gotchas in the YAML docs.
```
Comment (samdoran) [inline lib/ansible/modules/files/lineinfile.py:161]: Fully quoted is accurate. This line can be double of single quoted.

**Review by samdoran — CHANGES_REQUESTED** [review]

**Comment by acozine** [comment]
@xdfil thank you for taking the time to contribute to the Ansible project. I'm not sure of the reasons behind this PR - did the examples in the current docs fail for you, or generate errors? If so, how? That information would be helpful here - it's harder to assess documentation changes without more information on why you're proposing those changes.

Since this particular PR is old, has several unresolved change requests, and is failing CI, I'm going to close it. If this specific change is something you want to pursue, please open a new PR with more background information. If you're interested in contributing to the Ansible docs generally, I invite you to join the Documentation Working Group - we'd welcome your help! See https://github.com/ansible/community/issues/389 for more information.

**Pull request closed without merging**